src/doc/trpl/the-stack-and-the-heap.md

```diff
@@ -1,3 +1,526 @@
 % The Stack and the Heap
 
-Coming Soon
+As a systems language, Rust operates at a low level. If you’re coming from a
+high-level language, there’s some aspects of systems programming that you may
+not be familiar with. The most important one is how memory works, with a stack
+and a heap. If you’re familiar with the terms ‘stack’ and ‘heap’, this chapter
+will be a refresher. If you’re not, you’ll learn about this more general
+concept, but with a Rusty focus.
```

Comment:
Should 'Rusty' be lowercase? It's not a real adjective but also no a proper noun.